Anchor.toml

```diff
@@ -0,0 +1,19 @@
+[programs.localnet]
+fusion = "Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS"
+
+[registry]
+url = "https://anchor.projectserum.com"
+
+[provider]
+cluster = "localnet"
+wallet = "~/.config/solana/id.json"
+
+[[test.genesis]]
+address = "metaqbxxUerdq28cj1RbAWkYQm3ybzjb6a8bt518x1s"
+program = "fusion/program/tests/deps/mpl_token_metadata.so"
```

Comment:
This doesn't seem correct as the built `.so` files get written to `./target/deploy` folder in the root, see [here](https://github.com/metaplex-foundation/metaplex-program-library/pull/33/files#diff-8fbc7010039c384550aad716b7faf9354fee2d7b8bb84c200bfff581587ad331R5)

Reply:
So you should not somehow put it into `test/deps`, but use the one we build.